charts/cinder-csi-plugin/templates/controllerplugin-rbac.yaml

```diff
@@ -187,6 +187,7 @@ kind: RoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
   name: csi-resizer-role-cfg
+  namespace: {{ .Release.Namespace }}
```

Comment:
just wondering , where this value come from, could you point?

Reply:
please check discussion at #1215 ,it doesn't make changes to the chart from function perspective but make the definition consistent to other definitions we added

Reply:
@jichenjc that I got it, Im just asking for knowledge purpose

Reply:
I tested to make sure it will not affect functions..  thanks for asking